### PR TITLE
[FIX] Pull to get files with latest version

### PR DESCRIPTION
**Pull request by TheRolfFR**
Now for a read on a locked file, pulls latest version from top.

Add to your this .env to test it out :
COMPLIBOT_GIT_USERNAME=CompliBot
COMPLIBOT_GIT_EMAIL=CompliBot@users.noreply.github.com
COMPLIBOT_GIT_JSON_REPO=/Compliance-Resource-Pack/JSON/

### CODE REVIEW

**Comment by RobertRR11** [comment]
If I understand correctly, this makes the bot pull the json's from github before writing to them, right?

But does it do that on all json's? If yes, then the mute feature couldn't work, because the timer would always be reset from the github version, because the bot doesn't push on every timer change.

File: helpers/fileHandler.js [hunk 1]
```diff
@@ -75,7 +204,7 @@ class FileHandler {
           if(this.isJson) res = JSON.parse(res)
 
           // yeah yeah store the release function
-          this._release = release
+          this._release = real
```
Comment (Juknum) [inline helpers/fileHandler.js:207]: is it intended? 👀

Comment (TheRolfFR) [reply helpers/fileHandler.js:207]: Only way to chain promise results for murex release from acquire

**Comment by TheRolfFR** [comment]
@RobertRR11 I added pull options on file handler and disabled it for moderation one : https://github.com/Compliance-Resource-Pack/Discord-Bot/pull/71/commits/0d5e611d2c368029e600b56a84654d2ae183d8f7

**Comment by Juknum** [comment]
Everything seems ok for me, I am adding it

**Pull request merged**